compiler/haskell-ide-core/src/Development/IDE/State/Shake.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE ExistentialQuantification  #-}
 {-# LANGUAGE TypeFamilies               #-}
 {-# LANGUAGE ConstraintKinds            #-}
+{-# LANGUAGE DuplicateRecordFields      #-}
```

Comment:
Is this really necessary?

Reply:
Yes, haskell-lsp exports record fields with the same name